src/components/SearchBar.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { Grid, TextField, InputAdornment } from "@material-ui/core";
+import SearchIcon from "@material-ui/icons/Search";
+
+//  * Search Questions Elements/Details *
```

Comment:
I am assuming these comments are for you to use later as a guide for building functionality.